src/ec/packages/ec-component-tag/docs/tag.md

```diff
@@ -0,0 +1,36 @@
+# ECL Twig - EC Tag component
+
+npm package: `@ecl-twig/ec-component-tag`
+
+```shell
+npm install --save @ecl-twig/ec-component-tag
+```
+
+## Tag
+
+### Parameters
+
+- "tag" (associative array) (default: 'predefined structure below')
+  - "path" (string) (default: '') - path/url for tag (if needed - only for type 'link')
+  - "type" (string) (default: '') - type of tag (can be 'link', 'button', 'removable')
+  - "label" (string) (default: '') - tag label
+- "default_icon_path" (string ) (default: '') - path for the icon image (need to render Icon component if tag is removable)
+- "extra_classes" (optional) (string) (default: '') Extra classes (space separated) for the icon
```

Comment:
typo: this is tag and not icon here (and next line too)

Reply:
I don't know what you mean. The variable `default_icon_path` defines here the path to the file with icons, which is required for the tag type `removable`. Do you mean to change the variable name and description or define this path in the structure of the tag object. e.g. 
`tag: { type: 'removable', icon_path: '/pathtoicon' }`?

Reply:
no it's just the comment for extra_classes and extra_attributes. It should be tag instead of icon (but I agree that it is minor)

Reply:
Ok I see my mistake, I will correct everything after merging with the EC Icon.

Reply:
Fixed in https://github.com/ec-europa/ecl-twig/pull/17/commits/069ce0527105026e409f5ea3e42803bc88e2268b